docker-compose.yml

```diff
@@ -21,9 +22,22 @@ services:
     #         POSTGRES_USER: postgres
     #         POSTGRES_PASSWORD: postgres
 
-    # redis:
-    #     container_name: cache
-    #     image: redis
-    #     restart: always
-    #     ports:
-    #     - 6379:6379
+    cache:
+        image: redis
+        container_name: cache
+        restart: always
+        ports:
+            - 6379:6379
+
+    project:
```

Comment:
remove this conf with the `docker init` onr.

Reply:
Refer back to https://github.com/bassiounix/aim-platform

Reply:
I will but not now

Reply:
Mark assignment @G0maa. Proper setup then merge.